src/Compilers/CSharp/Test/Syntax/Parsing/SwitchExpressionParsingTests.cs

```diff
@@ -4796,4 +4796,60 @@ obj switch
         }
         EOF();
     }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/72720")]
+    public void TestConditionalExpressionAsPattern()
+    {
+        UsingExpression("""
+            obj switch
+            {
+                (flag ? a : b) => 1
```

Comment:
can you have a test for switch statements as well?

Reply:
They are already in the codebase. See commit https://github.com/dotnet/roslyn/pull/72805/commits/3a6f3bd8fab908d5d07767a2f423ad6580c14a1a, I've changed one of those tests back to what it was. The parenthesized case is exactly below it

Reply:
Here they are in `main`:
https://github.com/dotnet/roslyn/blob/93d19c2e18409d849860569be30128b2077f2ce8/src/Compilers/CSharp/Test/Syntax/Parsing/PatternParsingTests.cs#L12585-L12690